app/src/main/java/com/github/sdpcoachme/MainActivity.kt

```diff
@@ -17,40 +18,107 @@ import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.platform.testTag
 import com.github.sdpcoachme.ui.theme.CoachMeTheme
 
+
 class MainActivity : ComponentActivity() {
+
+
+    private lateinit var db: Database
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        db = (application as CoachMeApplication).database
         setContent {
             CoachMeTheme {
-                GreetingForm()
+                Column(
+                    modifier = Modifier.fillMaxSize(),
+                    verticalArrangement = Arrangement.Center,
+                    horizontalAlignment = Alignment.CenterHorizontally
+                ) {
+                    FirebaseForm()
+                    GreetingForm()
+                }
+            }
+        }
+    }
+
+    //TODO put test tags in separate class
+
+
+    /**
+     * A form for testing the database
+     */
+    @Composable
+    fun FirebaseForm() {
+        var phoneText by remember { mutableStateOf("") }
+        var emailText by remember { mutableStateOf("") }
+        Column( //modifier = Modifier.fillMaxSize(),
```

Comment:
You might want to remove those commented modifiers if the design works well without them. Otherwise, maybe a small describtion about when to uncomment those lines would be useful.